Image processing/image-resizer-filter-tool/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify, send_from_directory
+from flask_cors import CORS
+from PIL import Image, ImageDraw, ImageFont
+import io
+import base64
+
+app = Flask(__name__, static_url_path='')

```

Comment:
Hey @Jivanjamadar ,
Since you are using a static url path, you may need to specify for users to set it to `/static` in the README.
I will approve after you update the Readme otherwise project works well.

Reply:
Changes has been made in REAME.md file.